src/Extensions/HideInCampaignsExtension.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
This was used in both assets - https://github.com/silverstripe/silverstripe-assets/pull/660

And elemental - https://github.com/silverstripe/silverstripe-elemental/pull/1283